module/applications/actor/base-sheet.mjs

```diff
@@ -90,7 +90,8 @@ export default class ActorSheetEd extends ActorSheet {
    */
   _onLegendPointHistoryEarned( event ) {
     event.preventDefault();
-    this.actor.legendPointHistoryEarned();
+    const actor = this.actor;
```

Comment:
This is an unnecessary assignment. You can just use `this.actor` as the parameter.